cross/git/Makefile

```
@@ -38,7 +39,13 @@ include ../../mk/spksrc.cross-cc.mk
 git_pre_configure:
 	@$(RUN) autoconf
 
+.PHONY: git_post_compile
+git_post_compile:
+	@$(MSG) Compiling contrib subtree
+	@$(RUN) make -C contrib/subtree/
```

I prefere to use the MAKE variable and to add MAKE_COMPILE_OPTIONS. This is not mandatory here, as there is no compilation but only script processing for making subtree (but might be for other contributions)

```suggestion
	@$(RUN) $(MAKE) -C contrib/subtree/ $(MAKE_COMPILE_OPTIONS)
```